### PR TITLE
Feature/project detail

### PR DESCRIPTION
**Pull request by jenna1k**

### CODE REVIEW

File: src/components/Project/ProjectRow.js [hunk 1]
```diff
@@ -13,7 +16,7 @@ export default function ProjectRow(props) {
           type='button'
           className='btn btn-default btn-fab btn-fab-mini btn-round'>
           <i className='material-icons'>edit</i>
-        </button>{' '}
+        </button>
```
Comment (laura-mso) [inline src/components/Project/ProjectRow.js]: Here it was on purpose to have distance between the buttons, please do not remove this

File: src/components/Project/ProjectRow.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+import {Link} from 'react-router-dom';
 
 export default function ProjectRow(props) {
   return (
     <tr>
       <td>{props.project.id}</td>
-      <td className='text-warning'>{props.project.title}</td>
+      <Link to='/project-detail' className='link'>
+        <td className='text-warning'>{props.project.title}</td>
+      </Link>
```
Comment (laura-mso) [inline src/components/Project/ProjectRow.js]:
```suggestion
        <td className='text-warning'>
        <Link to='/' className='text-warning'>
          {props.project.title}
        </Link>
      </td>
```
Put the Link inside the td to keep the layout

**Review by laura-mso — CHANGES_REQUESTED** [review]

**Review by laura-mso — APPROVED** [review]

File: src/components/Project/ProjectRow.js [hunk 3]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 export default function ProjectRow(props) {
   return (
     <tr>
       <td>{props.project.id}</td>
-      <td className='text-warning'>{props.project.title}</td>
+      <td className='text-warning'>
+        <Link to='/project-detail' className='link'>
```
Comment (laura-mso) [inline src/components/Project/ProjectRow.js]:
```suggestion
        <Link to='/project-detail' className='link text-warning'>
```

**Review by laura-mso — CHANGES_REQUESTED** [review]

**Review by wilsoncooked — DISMISSED** [review]

**Review by wilsoncooked — COMMENTED** [review]
Looks great! However, on a large screen, there are gaps on the right side. I see it says the main container is col-md-10 where should it be col-md-12? Same with some components inside to then fit the 12 width?

File: src/App.js [hunk 4]
```diff
@@ -18,6 +19,7 @@ function App() {
           <Route path='/dashboard' component={Dashboard} />
           <Route path='/login' component={Login} />
           <Route path='/tables' component={Tables} />
+          <Route path='/project-detail' component={ProjectDetail} />
```
Comment (aliashkevich) [inline src/App.js]: I think it's better to have dynamic route smth as 'admin-panel.com/projects/:id' where :id is Project id

File: src/components/Project/ProjectDetailCard.js [hunk 5]
```diff
@@ -0,0 +1,523 @@
...
+    <div className='main-panel'>
+      <div className='content'>
+        <div className='container-fluid'>
+          <div className='row'>
+            <div className='col-md-10'>
+              <div className='card'>
+                <div className='card-header card-header-warning'>
+                  <h4 className='card-title'>Lesewert 2018</h4>
+                  <p className='category'>
+                    Project ID: <span>2018-33</span>
```
Comment (aliashkevich) [inline src/components/Project/ProjectDetailCard.js:14]: Need to consume API and use React, as I can see now it's fully static

File: src/components/Project/ProjectRow.js [hunk 6]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 export default function ProjectRow(props) {
   return (
     <tr>
       <td>{props.project.id}</td>
-      <td className='text-warning'>{props.project.title}</td>
+      <td className='text-warning'>
+        <Link to='/project-detail' className='text-warning'>
```
Comment (aliashkevich) [inline src/components/Project/ProjectRow.js]: Also, if possible, please add id into route.

**Review by aliashkevich — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 4 above)
Comment (aliashkevich) [reply src/App.js]: Postponed https://trello.com/c/BgfSPOet

File: src/components/Project/ProjectDetailCard.js (hunk 5 above)
Comment (aliashkevich) [reply src/components/Project/ProjectDetailCard.js]: Postponed https://trello.com/c/BgfSPOet

File: src/components/Project/ProjectRow.js (hunk 6 above)
Comment (aliashkevich) [reply src/components/Project/ProjectRow.js]: Postponed https://trello.com/c/BgfSPOet

File: src/App.js (hunk 4 above)
Comment (jenna1k) [reply src/App.js]: https://trello.com/c/fpnoYKTU

File: src/components/Project/ProjectDetailCard.js (hunk 5 above)
Comment (jenna1k) [reply src/components/Project/ProjectDetailCard.js:14]: https://trello.com/c/fpnoYKTU

File: src/components/Project/ProjectRow.js (hunk 6 above)
Comment (jenna1k) [reply src/components/Project/ProjectRow.js]: https://trello.com/c/fpnoYKTU

**Review by aliashkevich — DISMISSED** [review]

**Comment by jenna1k** [comment]
created new ticket for consuming API [check trello](https://trello.com/c/fpnoYKTU) and [anastasiya's comment](https://github.com/aliashkevich/AdminPanel/pull/20#discussion_r294221562)

File: src/App.js (hunk 4 above)
Comment (jenna1k) [reply src/App.js]:
```js
<Route exact path='/projects' component={Project} />
<Route path='/projects/:id' component={ProjectDetail} />
```
Thanks, I updated route!

**Review by laura-mso — APPROVED** [review]

File: src/components/Project/ProjectRow.js (hunk 6 above)
Comment (jenna1k) [reply src/components/Project/ProjectRow.js]:
```js
<Link to={`/projects/${props.project.id}`} className='text-info'>
          {props.project.title}
        </Link>
```
I passed `id` to `<Link>`

**Review by aliashkevich — APPROVED** [review]

**Pull request merged**